presto-main/src/main/java/com/facebook/presto/sql/planner/RelationPlanner.java

```diff
@@ -133,6 +136,16 @@
         this.subqueryPlanner = new SubqueryPlanner(analysis, symbolAllocator, idAllocator, lambdaDeclarationToSymbolMap, metadata, session);
     }
 
+    private static String subQueryNotSupportedError(Node node)
+    {
+        String formatString = "correlated subquery in given context is not supported";
```

Comment:
The error message should match exactly in `CheckSubqueryNodesAreRewritten` and `SemanticExceptions`. Otherwise the tests will fail.
nit: no need to call it `formatString`. `errorMessage` should be fine.